docs/projects/ocp/index.md

```diff
@@ -0,0 +1,204 @@
+---
+sidebar_position: 1
+---
+# OPA Control Plane
+
+
+OPA Control Plane (OCP) simplifies how you manage policies for your OPA
+deployments. It provides a centralized management system to control how OPAs
+receive the policies and data they need to make decisions. OCP provides:
+
+* **Git-based Policy Management.** Build bundles based on Rego from multiple Git
+  repositories and implement environment promotion strategies natively with Git.
+* **External Datasources.** Fetch and bundle external data required by your
+  policies build-time using HTTP push and pull datasources.
+* **Highly-Available & Scalable Bundle Serving.** Distribute bundles to cloud
+  object storage like AWS S3, Google Cloud Storage, or Azure Blob Storage and
+  ensure your OPAs can quickly and reliably serve policy decisions.
+* **Global and hierarchical policies.** Enforce organization-wide rules by
+  defining global policies that get injected into bundles at build-time based on
+  label selectors. Global policies can override other policies based on custom
+  conflict resolution logic written in Rego.
+
+# Learn More
+
+* [Deploy as a service](./guide-deploy-as-a-service.md) - Run OCP as a standalone service in Kubernetes
+* [Concepts](./concepts.md) - Learn how OCP works
+* [Configuration](./configuration.md) - Learn how to configure the server
+* [API Reference](./api-reference.md) - Learn about the OCP REST API
+* [Authentication](./authentication.md) - Learn how to secure the server API
+
+# Kick the tires
+
+Follow this section to get a quick example running on your laptop. By following
+these instructions, you will be able to:
+
+* Install OCP on your local machine.
+* Define a basic bundle with a test policy.
+* Use OCP to build the bundle
+* Configure OPA to use the OCP build bundle
+* Test the policy's enforcement and observe its effects.
+
+This example is designed for rapid iteration and learning, making it ideal for new users who want to understand OCP's fundamental concepts and operational flow in a controlled, personal setting. We'll focus on simplicity and clarity, ensuring that each step is easy to follow and the outcomes are immediately visible.
+
+## 1. Install binary
+
+Install the opactl tool using one of the install methods listed below.
+
+## 2. Define a bundle
+
+The bundle is defined by a configuration file normally in the `config.d` directory.  More details can be found in the [Concepts](#concepts) section, but for now lets use this configuration.  In your working directory add the following to `./config.d/hello.yaml`
+
+```yaml
```

Comment:
Odd that these produce no syntax highlighting. `shell` doesn't, either. I would have thought these would be missing based on the preview -- but they're here? 🤔

Reply:
Hmm, I am unsure if `sh` is the correct thing to use.